nginx.conf

```diff
@@ -27,11 +27,13 @@ http {
     server_tokens off;
 
     gzip on;
+    gzip_buffers 64 8k; # http://nginx.org/ru/docs/http/ngx_http_gzip_module.html#gzip_buffers
```

Comment:
Why do you use 64/8k buffers?

Reply:
According to the description in nginx, I assumed that the greater the number of gzip buffers, the faster nginx will be able to respond to the client. It's better to have more small buffers available than a few huge buffers.